post_results.py

```diff
@@ -84,18 +95,14 @@ def main():
             if msgtype == 'errors' or msgtype == 'failures' or msgtype == 'timeouts':
                 if 'test_extraction' in testcase['@classname']:
                     test_groups['clowder'] += 1
-                elif 'test_conversion' in testcase['@classname']:
-                    test_groups['polyglot'] += 1
-                elif 'test_input_output_graph' in testcase['@classname']:
-                    test_groups['polyglot'] += 1
-                else:
-                    test_groups['fence'] += 1
 
             log[msgtype].append(logmsg)
 
         mongoid = report_mongo(host, test_groups, elapsed_time, log)
         report_console(host, test_groups, elapsed_time, log, mongoid)
```

Comment:
what if we disable console in notifications?

Reply:
done